test/Main.hs

```diff
@@ -82,6 +83,7 @@ main = do
     , ("Feature.DeleteSpec"       , Feature.DeleteSpec.spec)
     , ("Feature.InsertSpec"       , Feature.InsertSpec.spec)
     , ("Feature.QuerySpec"        , Feature.QuerySpec.spec)
+    , ("Feature.RpcSpec"          , Feature.RpcSpec.spec)
```

Comment:
I don't see where the indentation is different, the first `,` has the same indentation as the others and the second `,` too.